test/it/feature-crud.js

```diff
@@ -48,20 +59,20 @@ describe('Feature Crud API', () => {
     beforeEach(async () => {
       // Disabling a BETA feature in a dev org throws 405 error
       // Hence we need a non-BETA feature for testing
-      await client.listFeatures().each((feature) => {
-        if (feature.stage.value !== 'BETA') {
-          firstFeatureInList = feature;
-          return false;
-        }
-      });
-
+      firstFeatureInList = await getFirstNonBetaFeature();
       if (firstFeatureInList) {
         initialStatus = firstFeatureInList.status;
       }
     });
     afterEach(async () => {
       if (firstFeatureInList) {
-        await firstFeatureInList.updateLifecycle(initialStatus === 'ENABLED' ? 'enable' : 'disable');
+        try {
+          await firstFeatureInList.updateLifecycle(initialStatus === 'ENABLED' ? 'enable' : 'disable');
+        } catch (err) {
+          if (err.status === 405 && err.status === 400) {
```

Comment:
@vijetmahabaleshwar-okta https://developer.okta.com/img/update-ssfeat-flowchart.png
looks like there are many cases that may result in 405/400, just catch it here to pass tests in bacon.